docs/2_development.md

```diff
@@ -183,7 +184,24 @@ below, with the relevant credentials and key values replaced.
   "twitterApiCredentials": {
     "appKey": "{TWITTER_APP_KEY}",
     "appToken": "{TWITTER_APP_TOKEN}",
-    "bearerToken": "{TWITTER_APP_BEARER_TOKEN}"
+    "bearerToken": "{TWITTER_APP_BEARER_TOKEN}",
+    "useEssentialOrElevatedV2": false
+  }
+}
+```
```

Comment:
```suggestion
```

```

Reply:
@kilemensi We have to escape the code section above. Why would we want to remove this?